src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/Resource.sql

```diff
@@ -19,6 +20,7 @@
 ALTER TABLE dbo.Resource SET ( LOCK_ESCALATION = AUTO )
 
 CREATE INDEX IX_ResourceTypeId_TransactionId ON dbo.Resource (ResourceTypeId, TransactionId) WHERE TransactionId IS NOT NULL ON PartitionScheme_ResourceTypeId (ResourceTypeId)
+CREATE INDEX IX_ResourceTypeId_HistoryTransactionId ON dbo.Resource (ResourceTypeId, HistoryTransactionId) WHERE HistoryTransactionId IS NOT NULL ON PartitionScheme_ResourceTypeId (ResourceTypeId)
```

Comment:
Will this be a long running index creation for existing customers?

Reply:
It will be long running, but indexes are created with online option, so there are no affect to customers.